Index: modules/Reports/ReportUtils.php
===================================================================
--- modules/Reports/ReportUtils.php	(revisión: 5217)
+++ modules/Reports/ReportUtils.php	(copia de trabajo)
@@ -90,6 +90,8 @@
 	} elseif (in_array($dbField->name,$report->ui101_fields) && !empty($value)) {
 		$entityNames = getEntityName('Users', $value);
 		$fieldvalue = $entityNames[$value];
+		if(strstr($dbField->name, "CobroPago"))
+			$fieldvalue = $value;		
 	} elseif( $fieldType == 'date' && !empty($value)) {
 		if($module == 'Calendar' && $field->getFieldName() == 'due_date') {
 			$endTime = $valueArray['calendar_end_time'];
Index: modules/Reports/ReportRun.php
===================================================================
--- modules/Reports/ReportRun.php	(revisión: 5217)
+++ modules/Reports/ReportRun.php	(copia de trabajo)
@@ -1723,10 +1723,36 @@
 		}
 
 		else {
-	 			if($module!=''){
+				if($module == "CobroPago")
+				{
+					global $adb;
+					
 	 				$focus = CRMEntity::getInstance($module);
 					$query = $focus->generateReportsQuery($module)
 								.$this->getRelatedModulesQuery($module,$this->secondarymodule)
+								.getNonAdminAccessControlQuery($this->primarymodule,$current_user);
+					$reluser=false;
+					foreach($this->referenceTableNames as $referenceTableName)
+					{
+						//rel_info: [0]->related tablename ; [1]->current module; [2]->id from the field wich is relating both tables; [3]->name of that field
+						$rel_info = explode(':',$referenceTableName);
+						if($rel_info[3] == 'comercialid' && !$reluser)
+						{
+							//getting primary key of the related table
+							$query_pk = $adb->query("SHOW KEYS FROM $rel_info[0] WHERE Key_name = 'PRIMARY'");
+							$pk = $adb->query_result($query_pk, 0 ,'column_name');
+							
+							//adding the left join sentence
+							$query .= "left join $rel_info[0] AS $rel_info[0]Rel$rel_info[1]$rel_info[2] on vtiger_cobropago.$rel_info[3] = $rel_info[0]Rel$rel_info[1]$rel_info[2].$pk ";
+							$reluser = true;
+						}
+					}
+					$query .= "where vtiger_crmentity.deleted=0 ";
+				}
+	 			else if($module!=''){
+	 				$focus = CRMEntity::getInstance($module);
+					$query = $focus->generateReportsQuery($module)
+								.$this->getRelatedModulesQuery($module,$this->secondarymodule)
 								.getNonAdminAccessControlQuery($this->primarymodule,$current_user).
 							" WHERE vtiger_crmentity.deleted=0";
 	 			}
